angelsbioprocessing/prototypes/technology/crop-farming.lua

```diff
@@ -57,16 +57,40 @@ data:extend(
         {
           type = "unlock-recipe",
           recipe = "swamp-garden-cultivating-a"
+        }
+      },
+      unit = {
+        count = 50,
+        ingredients = {
+          {type = "item", name = "automation-science-pack", amount = 1},
+          {type = "item", name = "logistic-science-pack", amount = 1}
         },
+        time = 30
+      }
+    },
+    {
+      type = "technology",
+      name = "gardens-3",
+      icon = "__angelsbioprocessing__/graphics/technology/seed-extractor-tech.png",
+      icon_size = 128,
+      order = "c-a",
+      prerequisites = {
+        "bio-farm-alien",
+        "chemical-science-pack"

```

Comment:
Fix for unit test failure:
```
angelsdev-unit-test: Starting unit test 006.
angelsdev-unit-test:     Technology "gardens-2" requires higher science packs than its prerequisites provide.
```
```suggestion
        "bio-farm-1",
        "bio-paper-1",
        "logistic-science-pack"
```